### ISSUE TITLE
Pulumi login file://<path> does not use path specified

### ISSUE THREAD

**Issue by erikj23** [pulumi/pulumi#9508]
### What happened?

Using the  [login documentation](https://www.pulumi.com/docs/reference/cli/pulumi_login/) I'm trying to create a local workspace *NOT* in my `home` folder. 

Running `pulumi login file://<path>` creates a folder in the home directory regardless of the `path` specified.

### Steps to reproduce

```
➜  ~ cd ~
➜  ~ rm -rf ~/.pulumi        
➜  ~ ls -al | grep pulumi    
➜  ~ pulumi login file:///tmp
Logged in to macp as erikmaldonado (file:///tmp)
➜  ~ ls -al | grep pulumi    
drwx------    4 erikmaldonado  staff    128 May  2 20:26 .pulumi
➜  ~ 
```

### Expected Behavior

I expect pulumi to instead use the passed in `path`.

### Actual Behavior

Pulumi creates the `.pulumi` folder in the `home` directory.

### Versions used

```
➜  ~ pulumi about
CLI          
Version      3.31.0
Go Version   go1.18.1
Go Compiler  gc

Host     
OS       darwin
Version  12.3.1
Arch     x86_64

Backend        
Name           macp
URL            file:///tmp
User           erikmaldonado
```

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mikhailshilkov** [comment]
Hi @erikj23
The `.pulumi` folder contains all things Pulumi: binaries, plugins, templates, etc. None of these are currently controlled with the backend URL parameter. Do you see the `tmp` folder created? Does it contain your state files?

**Comment by Frassle** [comment]
N.B You can change the location of the .pulumi folder by setting the environment variable PULUMI_HOME

**Comment by erikj23** [comment]
@Frassle that did what I was trying to accomplish, thank you. For future users, is there documentation that details this option?
@mikhailshilkov after the modification recommended by Frassle, the `.pulumi` folder did get moved to a new location, `/tmp`. I had to also set `PULUMI_CREDENTIALS_PATH=/tmp` as well or it would not work. As a test I set the backend URL to /tmp/pulumi, different from the pulumi home folder. After a resource generation /tmp/pulumi contained a single json cache folder with the stack name. The `/tmp/.pulumi` contained the actual built stack.
